strands_navigation_msgs/msg/Edge.msg

```diff
@@ -6,9 +6,8 @@ string map_2d
 bool use_default_recovery
 string[] monitor_recovery_pairs_names
 strands_navigation_msgs/DynClassLoaderDef[] monitor_states
-strands_navigation_msgs/DynClassLoaderDef[] recovery_state_machines
 bool use_default_nav_recovery
```

Comment:
yes It was accident, I was doing some testing changing the messages for the migration and I left it changed sorry, however this takes me to the next issue, how about we put all this things (except for the booleans) in a JSON/Yaml string, wouldn't it be more comprehensible?